src/main/java/hudson/plugins/distfork/DistForkCommand.java

```diff
@@ -93,12 +81,12 @@ protected int run() throws Exception {
         if(commands.isEmpty())
             throw new CmdLineException(null, "No commands are specified");
 
-        Jenkins j = Jenkins.getInstance();
+        Jenkins j = Jenkins.get();
 
         Label l = null;
         if (label!=null) {
             l = j.getLabel(label);
-            if(l.isEmpty()) {
+            if(l == null || l.isEmpty()) {
```

Comment:
`Jenkins.getLabel()`  is marked as `CheckForNull` so this addresses a spotbugs violation